frontend/src/metabase-types/api/dataset.ts

```diff
@@ -59,9 +67,9 @@ export type JsonQuery = DatasetQuery & {
 
 export interface Dataset {
   data: DatasetData;
-  database_id: DatabaseId;
+  database_id?: DatabaseId;
   row_count: number;
-  running_time: number;
+  running_time?: number;
```

Comment:
Are these really optional?

Reply:
Thank you for pointing this out. I cannot remember why I did this. Reverting it.